packages/data-migration/src/cli/LogStream.ts

```diff
@@ -0,0 +1,76 @@
+import CloudWatchLogs from "aws-sdk/clients/cloudwatchlogs";
+
+const cache = new Map<string, LogStream>();
+
+export class LogStream {
+    private readonly logGroupName: string;
+    private readonly logStreamName: string;
+    private readonly cloudWatchLogs: CloudWatchLogs;
+    private nextPage: string | undefined;
+
+    private constructor(logGroupName: string, logStreamName: string) {
+        this.logGroupName = logGroupName;
+        this.logStreamName = logStreamName;
+        this.cloudWatchLogs = new CloudWatchLogs();
+    }
+
+    getLogStreamLink() {
+        const replacements = [
```

Comment:
Where did you find what needs to be replaced?
Wouldnt the encodeURIComponent help?

Reply:
😂 I had to look at CloudWatch URLs in the browser. And they're not using encodeURIComponent, but some custom encoding here, see the difference:

encodeURIComponent:
```%2Faws%2Flambda%2Fwby-data-migration-8a6968a%2Flog-events%2F2023%2F08%2F24%2F%5B%24LATEST%5De41d9ccf18fe487c9599ad4274ceef3f```

AWS encoding:
```
$252Faws$252Flambda$252Fwby-data-migration-8a6968a/log-events/2023$252F08$252F24$252F$255B$2524LATEST$255De41d9ccf18fe487c9599ad4274ceef3f
```